### PR TITLE
test: add client performance/benchmark test

### PR DESCRIPTION
**Pull request by rhajek**
## Proposed Changes

This PR adds client benchmarking test that validates the client writes in multithreaded load.

## Checklist

- [ ] CHANGELOG.md updated
- [ ] Rebased/mergeable
- [ ] A test has been added if appropriate
- [ ] `mvn test` completes successfully
- [ ] Commit messages are in [semantic format](https://seesparkbox.com/foundry/semantic_commit_messages)
- [ ] Sign [CLA](https://www.influxdata.com/legal/cla/) (if not already signed)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -197,6 +204,10 @@ workflows:
       - tests-java:
           name: jdk-8-nightly
           influxdb-image: "quay.io/influxdb/influxdb:nightly"
+      - tests-java:
+          name: client-benchmark
+          influxdb-image: "quay.io/influxdb/influxdb:nightly"
```
Comment (bednar) [inline .circleci/config.yml:209]: Please use stable image:

```suggestion
```

File: client/src/test/java/com/influxdb/client/ITBenchmarkTest.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+        bucketsApi.findBuckets()
+            .stream()
+            .filter(bucket -> bucket.getName().endsWith("-IT"))
+            .forEach(bucket -> {
+                console("delete bucket: " + bucket.getName());
+                bucketsApi.deleteBucket(bucket);
+            });
+
+        bucket = bucketsApi.createBucket("benchmark-IT", findMyOrg());
```
Comment (bednar) [inline client/src/test/java/com/influxdb/client/ITBenchmarkTest.java:66]: The bucket name has to be unique:

```suggestion
        bucket = bucketsApi.createBucket(generateName("benchmark"), findMyOrg());
```

File: client/src/test/java/com/influxdb/client/ITBenchmarkTest.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    void setUp() {
+
+        bucketsApi = influxDBClient.getBucketsApi();
+        OrganizationsApi organizationsApi = influxDBClient.getOrganizationsApi();
+
+        organizationsApi.findOrganizations()
+            .stream()
+            .filter(org -> org.getName().endsWith("-IT"))
+            .forEach(organizationsApi::deleteOrganization);
+
```
Comment (bednar) [inline client/src/test/java/com/influxdb/client/ITBenchmarkTest.java:57]: Delete organization is not necessary

```suggestion
```

File: pom.xml [hunk 4]
```diff
@@ -112,7 +112,7 @@
         <dependency.okhttp3.version>4.7.2</dependency.okhttp3.version>
         <dependency.gson.version>2.8.8</dependency.gson.version>
 
-        <plugin.surefire.version>2.22.2</plugin.surefire.version>
+        <plugin.surefire.version>3.0.0-M3</plugin.surefire.version>
```
Comment (bednar) [inline pom.xml:115]: Please use latest milestone version:

```suggestion
        <plugin.surefire.version>3.0.0-M5</plugin.surefire.version>
```

**Review by bednar — CHANGES_REQUESTED** [review]
Thanks for your PR 👍

There are a few requirements that must be be satisfy before we accept the PR:

File: client/src/test/java/com/influxdb/client/ITBenchmarkTest.java (hunk 3 above)
Comment (rhajek) [reply client/src/test/java/com/influxdb/client/ITBenchmarkTest.java]: Delete of test organizations removed.

**Review by bednar — APPROVED** [review]
LGTM 👍

**Pull request merged**